src/server/Backend.js

```diff
@@ -0,0 +1,31 @@
+class Backend {
+
+    constructor() {
+
+    	this.apiUrlEndpoint = '/bike-shop-api/';
+
+        this.apiUrlMap = new Map();
+        this.apiUrlMap.set('high-all', 'high-low-all-products');
+        this.apiUrlMap.set('low-all', 'low-high-all-products');
+        this.apiUrlMap.set('high-accessories', 'high-low-accessories-products');
+        this.apiUrlMap.set('low-accessories', 'low-high-accessories-products');
+        this.apiUrlMap.set('high-bikes', 'high-low-bikes-products');
+        this.apiUrlMap.set('low-bikes', 'low-high-bikes-products');
+        this.apiUrlMap.set('high-components', 'high-low-components-products');
+        this.apiUrlMap.set('low-components', 'low-high-components-products');
+    }
```

Comment:
We should get together and spec out the API.  I suspect it will look quite different from what's on the template now! :)

Reply:
Created this map as a placeholder, but this will soon be removed in favor of the final data service.
As for the API definition, I found that this one would support Wompmobile with minor changes:

    /api/categories: Accepting "categoryId" and "sort".
    /api/product: Accepting "productId" param, and should return the associated JSON.

In any case, It would be interesting to define the API with independence of the actual data, but I'm afraid that until we arrive to a conclusion on which service to use for the product data.